recipes/aiohttp-client-cache/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "aiohttp-client-cache" %}
+{% set version = "0.3.0" %}
+{% set hash_val = "d6feafb9924cfd40159f80fd4cd5f39794ff6b90f209bca190a5432d64892cab" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_val }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```

Comment:
I know it built successfully somehow, but this should probably list `poetry` here as well

Reply:
The current version on pypi ([v0.3.0](https://pypi.org/project/aiohttp-client-cache/0.3.0/)) was built with plain old [setup.py](https://github.com/JWCook/aiohttp-client-cache/blob/v0.3/setup.py).

I just updated the project to use poetry, but haven't made a new release yet. I was planning to update the recipe after releasing the next version. Or is there a better way to handle that?

Reply:
Ah, I missed that--I just looked at the repo. That explains why it works. 😄 You can update the recipe when you bump the version in the feedstock.